bash2048.sh

```diff
@@ -262,6 +264,70 @@ function reload_game {
   index_max=board_size-1
 }
 
+# print the content of the highscores file
+function show_highscores {
```

Comment:
there should be a check if highscore file exists, otherwise this will just result in an error if no hiscore file is present.

Reply:
Yes you're right :+1: